app/src/components/HolidayList.vue

```diff
@@ -13,8 +13,24 @@
 <script lang="ts">
 import { defineComponent } from "vue";
 import { useCountryStore } from "@/stores/CountryStore";
+import {
+  IonList,
+  IonListHeader,
+  IonRow,
+  IonCard,
+  IonCardTitle,
+  IonCardContent
+} from '@ionic/vue'
 
 export default defineComponent({
+  components: {
```

Comment:
The issue was that the imports and components weren't there so no wonder it struggled. I think serving with the ionic plugin probably had the node_modules in context so could resolve the components on the fly but the build never had them packaged.